charts/astronomer/values.yaml

```diff
@@ -73,7 +73,8 @@ houston:
   regenerateCaEachUpgrade: false
   # Houston datastore
   backendSecretName: ~
-  backendConnection: {}
+  backendConnection:
+    {}
```

Comment:
This was automatically done by `prettier -w`. I don't necessarily like it, but i like auto-formatting because it means we don't have to argue about style prefs.

Reply:
https://prettier.io/ ?

Reply:
i like such tools too, only i prefer to add automatic check on repo so if somebody next time create PR it fails build after running `make lint` or other way.

Reply:
If you're suggesting we enforce auto-formatting in this repo or any other I will 100% support that. Usually I do this with pre-commit.

- https://github.com/astronomer/ci-images/blob/main/.circleci/config.yml#L5-L6
- https://github.com/astronomer/ci-images/blob/main/.pre-commit-config.yaml#L6-L8

Reply:
yeah, it's the one that we recently switched off of for a bunch of other code bases.